client/blocks/editor-checkout-modal/index.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import wp from 'lib/wp';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { StripeHookProvider } from 'lib/stripe';
+import { fetchStripeConfiguration } from 'my-sites/checkout/composite-checkout/payment-method-helpers';
+import CompositeCheckout from 'my-sites/checkout/composite-checkout/composite-checkout';
+import { getSelectedSite } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const wpcom = wp.undocumented();
+
+class EditorCheckoutModal extends Component {
+	static propTypes = {
+		site: PropTypes.object,
+		isOpen: PropTypes.bool,
+		onClose: PropTypes.func,
+		cartData: PropTypes.object,
+	};
```

Comment:
I think PropTypes aren't needed in TypeScript files :D

Reply:
Perils of copy & paste!